vassal-app/src/main/java/VASSAL/counters/Decorator.java.orig

```diff
@@ -0,0 +1,736 @@
+/*
+ *
+ * Copyright (c) 2000-2012 by Rodney Kinney, Brent Easton
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License (LGPL) as published by the Free Software Foundation.
```

Comment:
oops, what's this guy doing in here?

Reply:
ACK! Detritus from a conflict-resolve. Somehow git likes to leave those lying around if I don't remember to manually delete them.

Fixed up.